engine/runtime-compiler/src/main/scala/org/enso/compiler/phase/ImportResolver.scala

```diff
@@ -46,7 +47,12 @@ final class ImportResolver(compiler: Compiler) extends ImportResolverForIR {
           )
           (ir, currentLocal)
         } catch {
-          case _: IOException =>
+          case ex: IOException =>
+            context.logSerializationManager(
+              Level.WARNING,
+              "Deserialization of " + module.getName() + " failed",
+              ex
```

Comment:
@hubertp, when this exception is logged, the stack trace isn't printed on the console. What's the recommended way of logging an exception with stacktrace via `context.logSerializationManager`?

Reply:
I don't think I'm surprised anymore. It looks like the included `application.conf` does not redefine the layout of logging, meaning it does not print exceptions.

Reply:
I will file a separate ticket to enable those. Right now the default and only appender is console. We could easily log those to the file but I'm still against printing those in the console.